### PR TITLE
feature/COR-1356-death-rate-page-adjustments

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard26**
## Summary

* updated 'Sterfte' (or 'Death rate') pages across NL, VR and GM to correctly archive (parts of) the pages
* updated GM layout to enhance the navigational sidebar with an 'Archived pages' section
* updated various JSON schemas to be marked as archived across NL, VR and GM
* generated data types based on new schema updates
* updated Sanity keys

### Screenshots
Any texts in the screenshots are not set in stone.

#### Before
<details>
<summary>Toggle before screenshots</summary>

COR-1356_before_NL
COR-1356_before_VR
COR-1356_before_GM

</details>

#### After
<details>
<summary>Toggle before screenshots</summary>

COR-1356_after_NL
COR-1356_after_VR
COR-1356_after_GM

</details>

### CODE REVIEW

File: packages/app/src/pages/landelijk/sterfte.tsx [hunk 1]
```diff
@@ -69,112 +72,16 @@ function DeceasedNationalPage(props: StaticProps<typeof getStaticProps>) {
     description: textNl.metadata.description,
   };
 
+  const hasActiveWarningTile = !!textShared.notification.message && !isEmpty(textShared.notification.message);
```
Comment (VWSCoronaDashboard28) [inline packages/app/src/pages/landelijk/sterfte.tsx:75]: Wouldn't just `!!textShared.notification.message` or `!isEmpty(textShared.notification.message)` suffice?

**Review by VWSCoronaDashboard28 — APPROVED** [review]
Approved but one small comment

**Review by VWSCoronaDashboard25 — APPROVED** [review]

File: packages/app/src/pages/landelijk/sterfte.tsx (hunk 1 above)
Comment (VWSCoronaDashboard26) [reply packages/app/src/pages/landelijk/sterfte.tsx]: Opted to go with just `!!textShared.notification.message` to keep the `boolean` check in place. This means we also got rid of `isEmpty`.

Will mark this conversation as resolved.

File: packages/app/schema/gm/__index.json [hunk 2]
```diff
@@ -35,8 +35,8 @@
     "static_values": {
       "$ref": "__static_values.json"
     },
-    "deceased_rivm": {
-      "$ref": "deceased_rivm.json"
+    "deceased_rivm_archived_20221231": {
+      "$ref": "deceased_rivm_archived_20221231.json"
```
Comment (VWSCoronaDashboard25) [inline packages/app/schema/gm/__index.json:39]: Why do we rename the files as well?

I don't think it's necessary for the changes in the backend. Not sure what is best though and consistency is a great argument, but is it actually preferable to have references to schemas labeled with `_archived_YYYYMMDD`? For me it feels just like extra error prone effort.

Same question for the title of that schema of which the file name was changed.

Comment (VWSCoronaDashboard26) [reply packages/app/schema/gm/__index.json:39]: Both BE and FE have agreed that these keys are to be archived as can be found in a comment in the associated ticket for this feature. Do note that that comment also refers to another ticket, which is the BE ticket for this feature change.

Newly generated data already takes care of these changes.

I do understand why you would say that using `_archived_YYYYMMDD` suffixes for schemas and their associated data is more prone to error, but then we (the broader organisation) would have to come up with a different way to 'mark' schemas and associated data as archived.

**Pull request merged**